execution_engine/src/runtime/mod.rs

```diff
@@ -2512,14 +2532,21 @@ where
     }
 
     /// Enforce group access restrictions (if any) on attempts to call an `EntryPoint`.
-    fn validate_group_membership(
+    fn validate_entry_point_access(
         &self,
         package: &ContractPackage,
+        name: &str,
         access: &EntryPointAccess,
     ) -> Result<(), Error> {
-        runtime_context::validate_group_membership(package, access, |uref| {
-            self.context.validate_uref(uref).is_ok()
-        })
+        match access {
+            EntryPointAccess::Public => Ok(()),
+            EntryPointAccess::Groups(groups) => {
+                runtime_context::validate_group_membership(package, groups, |uref| {
+                    self.context.validate_uref(uref).is_ok()
+                })
+            }
+            EntryPointAccess::Abstract => Err(Error::NoSuchMethod(name.to_string())),
```

Comment:
Feels like we should add a bespoke error here, `Error::InvalidEntryPointAccess` , I feel like no such method is sightly misleading

Reply:
Added `AbstractMethod` variant (that follows language of `NoSuchMethod`)